code/modules/arcade/crane.html

```diff
@@ -204,22 +205,19 @@
 		window.oRequestAnimationFrame ||
 		window.msRequestAnimationFrame ||
 		function(callback){
-			window.setTimeout(callback, 1000 / 30);
+			window.setTimeout(callback, 33.333);	//1000 / 30
 		};
 	})();
 
 	function animate(){
 		//console.log("animate called");
 		crane.Update();
-		
-		for(var i = 0; i< prizes.length; i++)
+		for(var i = 0; i < 5; i++){
```

Comment:
Why is this just 5 now, and not `prizes.length`?

Reply:
There was another for loop in there that used just 5 so I thought why not, probably better to of changed both of them to length...

Reply:
One of them is initializing the list, I believe, though it might be helpful in that case to use a define. Here it doesn't make sense because it decreases maintainability.